modules/jaggery-apps/user-dashboard/dashboard/samlsso.jag

```diff
@@ -7,8 +7,14 @@ var sso = require("sso");
 var sessionId = session.getId();
 var encodedSAMLRequest;
 var postUrl;
-var postUrl = samlConfig['SAML.IdPUrl'];
-var issuer = samlConfig['SAML.IssuerID'];
+var issuer;
+if(samlConfig!=null){
+    postUrl = samlConfig['SAML.IdPUrl'];
+    issuer = samlConfig['SAML.IssuerID'];
+}
```

Comment:
why not have an else condition and do the redirect here instead of having a new if condition ?